tests/playwright/sanity/modules/v4-tests/atomic-repeaters-display.test.ts

```diff
@@ -0,0 +1,49 @@
+import { parallelTest as test } from '../../../parallelTest';
+import WpAdminPage from '../../../pages/wp-admin-page';
+import EditorPage from '../../../pages/editor-page';
+import { BrowserContext, Page, expect } from '@playwright/test';
+
+test.describe( 'Atomic repeaters display @atomic-widgets', () => {
+	let editor: EditorPage;
+	let wpAdmin: WpAdminPage;
+	let context: BrowserContext;
+	let page: Page;
+
+	test.beforeAll( async ( { browser, apiRequests }, testInfo ) => {
+		context = await browser.newContext();
+		page = await context.newPage();
+		wpAdmin = new WpAdminPage( page, testInfo, apiRequests );
+		await wpAdmin.setExperiments( {
+			e_atomic_elements: 'active',
+		} );
+	} );
+
+	test.afterAll( async () => {
+		await wpAdmin.resetExperiments();
+		await context.close();
+	} );
+
+	const repeaterControls = [ 'filter', 'Backdrop filter', 'transform', 'Box shadow', 'transitions' ];
+
+	repeaterControls.forEach( ( contorl ) => {
```

Comment:
I remember Dmytro suggesting to avoid Array methods as they behave unexpectedly in async functions
```suggestion
	for ( const control of repeaterControls ) {
```